pkgs/by-name/ne/neocities-cli/package.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, bundlerApp
+, bundlerUpdateScript
+, installShellFiles
+}:
+
+let
+  ttBundlerApp = bundlerApp rec {
```

Comment:
not sure why not just use bundlerApp as final derivation ?

Reply:
Yes, I just realize I can cut in half the size of the file quite easily.